scripts/dts/edtsdatabase.py

```diff
@@ -127,6 +135,21 @@ def _update_device_compatible(self, device_id, compatible):
             self._edts['compatibles'][compatible].append(device_id)
             self._edts['compatibles'][compatible].sort()
 
+    def _update_device_type(self, device_id, device_type):
+        if device_type not in self._edts['device-types']:
+            self._edts['device-types'][device_type] = list()
+        if device_id not in self._edts['device-types'][device_type]:
+            self._edts['device-types'][device_type].append(device_id)
+            self._edts['device-types'][device_type].sort()
+
+    def insert_device_type(self, compatible, device_type):
```

Comment:
Remove - already accomplished by insert.

Reply:
I've chosen a different device-type dict as the one you proposed.
I've chosen to display: dict(device-type : sorted list(compatible)) instead of a sorted list of device_id.
Aim was to make it easier to access to get device-type information per compatible (for DTS_ #define's generation), as well as device-type list for CI.
So I need to inject (device-type, compatible) couple, which I can't using insert.